### DISCUSSION TITLE
Canonical .winmd file?

### DISCUSSION THREAD

**Discussion by timsneath** [microsoft/win32metadata#409] [Q&A] [answered]
The `README.md` says: 

> If you'd like to browse the metadata to see what we're emitting, download the NuGet package and load the included winmd file in ILSpy.

Is that the canonical source for an official metadata file? Or is the [file in scripts\BaselineWinmd](https://github.com/microsoft/win32metadata/blob/master/scripts/BaselineWinmd/10.0.19041.5/Windows.Win32.winmd) the latest generated winmd? Or should I continue to generate my own one using the `DoAll.ps1` script?

My reason for asking is that I include a copy in [my Dart package](https://github.com/timsneath/winmd) that matches my WinMD parser, so that I can be sure that the parser and the Win32 source are compatible and tested together successfully. Right now I'm including a file I generate on my own machine, but as the generated file starts to stabilize, I might prefer to use a canonical Microsoft-supplied version.

**Comment by kennykerr** [comment, marked as answer]
Yes, I always extract the winmd file from the latest nuget package.
